src/utils/decorators/input-validation/input-validation.js

```diff
@@ -173,7 +177,16 @@ const InputValidation = (ComposedComponent) => {
        * @property
        * @type {Array}
        */
-      info: PropTypes.array
+      info: PropTypes.array,
+
+      /**
+       * Number which sets timing of when the message will disappear
```

Comment:
💄 Cosmetic thing but it would help to state here that the time is expected to be in miliseconds